### PR TITLE
feat: add gas used

### LINKED ISSUE #71: Display proper gas used on block-explorer
if we can either display the proper gas used in the block-explorer-v2, or just remove it.

### LINKED ISSUE #72: Remove maturity field on tx explorer

### PR DESCRIPTION
**Pull request by LuizAsFight**
- added real `gasUsed` value to transactions page
- hide maturity field
- change `DECIMAL_UNITS=9`  to support `u64`
- hided blocks/transactions list column `X time ago`, because backend is returning always same date `1/1/1970`

Closes #71 
Closes #72 
Ref.: #73

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "npm": ">=8.0.0"
   },
   "scripts": {
-    "start": "craco start",
+    "start": "BROWSER=brave craco start",
```
Comment (luizstacio) [inline package.json:9]: Why we need the `BROWSER` here?

**Review by luizstacio — APPROVED** [review]
Just one question

**Review by adlerjohn — CHANGES_REQUESTED** [review]
Wait a second. I'm getting something like

```
Gas Limit: 0.000705042
Gas Used: 0.000000002
```

That's not right. Gas limit and gas used are integers. I think you're computing the amount of ETH limit/used, i.e. gas price * gas limit/used.

File: src/pages/TransactionPage/index.tsx [hunk 2]
```diff
@@ -47,14 +48,25 @@ import {
...
 
   if (!tx) return null;
 
+  const transactionFee = calculateTransactionFee({
+    bytePrice: +tx.bytePrice,
+    rawPayload: tx.rawPayload,
+    witnesses: tx.witnesses,
+    gasPriceFactor: +(chains[0].consensusParameters?.gasPriceFactor || 0),
+    gasPrice: +tx.gasPrice,
+    gasLimit: +tx.gasLimit,
```
Comment (adlerjohn) [inline src/pages/TransactionPage/index.tsx:67]: So, this is computing the _maximum_ fee the transaction will pay, not how much it actually paid. To compute that, you need to know the gas used (which was part of the original intention of this issue).

To get gas used, you need to read the last receipt of the transaction, and read the `gas_used` field: https://github.com/FuelLabs/fuel-specs/blob/204136996a795811907b472c88966dc09867e2a5/specs/protocol/abi.md#scriptresult-receipt

**Review by adlerjohn — CHANGES_REQUESTED** [review]

**Comment by adlerjohn** [comment]
Ah I see. After that it works. It just wasn't documented I need to do that.

File: src/pages/TransactionPage/index.tsx [hunk 3]
```diff
@@ -88,8 +101,8 @@ export default function TransactionPage() {
               <RowValueColumn>{parseToFormattedNumber(tx.gasLimit)}</RowValueColumn>
             </TransactionDataRow>
             <TransactionDataRow>
-              <RowKeyColumn>Gas Used:</RowKeyColumn>
-              <RowValueColumn>0</RowValueColumn>
+              <RowKeyColumn>Transaction fee:</RowKeyColumn>
+              <RowValueColumn>{parseToFormattedNumber(transactionFee)}</RowValueColumn>
```
Comment (adlerjohn) [inline src/pages/TransactionPage/index.tsx:105]: Goods looks like this works!. Can you add a row now for the gas used?

Comment (LuizAsFight) [reply src/pages/TransactionPage/index.tsx:105]: of course !

**Comment by LuizAsFight** [comment]
@adlerjohn 

I also added 
- `bytePrice` row
- `ETHER` sufixes to other rows

File: src/pages/TransactionPage/index.tsx [hunk 4]
```diff
@@ -76,23 +89,35 @@ export default function TransactionPage() {
...
               <RowValueColumn>
                 {parseToFormattedNumber(tx.gasPrice)} {BASE_COIN_NAME}
               </RowValueColumn>
             </TransactionDataRow>
+            <TransactionDataRow>
+              <RowKeyColumn>Byte Price:</RowKeyColumn>
+              <RowValueColumn>
+                {parseToFormattedNumber(tx.bytePrice)} {BASE_COIN_NAME}
+              </RowValueColumn>
+            </TransactionDataRow>
```
Comment (adlerjohn) [inline src/pages/TransactionPage/index.tsx:107]: Byte price is being removed in the next release, so don't even bother with it. It'll just confuse people. https://github.com/FuelLabs/fuel-specs/pull/333

**Review by adlerjohn — APPROVED** [review]
good work

**Pull request merged**